app/jobs/regular/process_bulk_invite_emails.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_dependency 'email/sender'
+
+module Jobs
+
+  class ProcessBulkInviteEmails < Jobs::Base
+
+    def execute(args)
+      bulk_invite_email_key = Invite::BULK_INVITE_EMAIL_KEY
```

Comment:
Looks like this variable is unused? Probably the constant too.

Reply:
Good catch, fixed!